airbyte-integrations/connectors/source-iterable/source_iterable/iterable_streams.py

```diff
@@ -47,6 +47,15 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
     def request_params(self, **kwargs) -> MutableMapping[str, Any]:
         return {"api_key": self._api_key}
 
+
+    def request_headers(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> Mapping[str, Any]:
+        headers = super().request_headers(stream_state=stream_state,
+                                          stream_slice=stream_slice, next_page_token=next_page_token)
```

Comment:
@ssbeefeater 
1. Can you confirm if earlier without overriding implementation of request_headers was doing only  what is done at line 54?
2. If yes for point 1, so what we have done is only adding apiKey header which is the fix.

Thanks for quick fix 👏

Reply:
1. Yeah class IterableStream extends class HTTPStream that includes this method for updating the headers

Reply:
i Also removed the API key from query params as is not needed any ore

Reply:
Please check if this is fine or ok to keep API Key in query params.